src/constants/productTierFeatures.tsx

```diff
@@ -0,0 +1,78 @@
+import CloudNative from "src/components/Icons/productTierFeatures/CloudNative";
+import Datadog from "src/components/Icons/productTierFeatures/Datadog";
+import NewRelic from "src/components/Icons/productTierFeatures/NewRelic";
+import Signoz from "src/components/Icons/productTierFeatures/Signoz";
+
+export const INTEGRATION_SCOPES = {
```

Comment:
Some variables declared here are not being used anywhere. Can you check and remove those?